README.md

```diff
@@ -172,3 +171,162 @@ Removed following and all unused discs from Azure Cloud.
 Removed following and all unused pips from Azure Cloud.
 ['test-axodawttrw-pip0']
 ```
+# How to run the cloudwash setup locally
```

Comment:
I think we should create a separate doc for this and refer the document source in README. This would be a very big addition in README doc itelf.